clash-lib/src/Clash/Util.hs

```diff
@@ -348,20 +289,6 @@ clogBase x y | x > 1 && y > 0 =
                 else Just (I# z1)
 clogBase _ _ = Nothing
 
--- | Determine whether two lists are of equal length
-equalLength
```

Comment:
Same here, why remove this when `equalLength xs ys` finishes earlier than `lenght xs == length ys`